src/components/Header/ProductCategory.jsx

```diff
@@ -0,0 +1,177 @@
+import { useState } from "react";
+import { Link } from "react-router-dom";
+import styles from "./ProductCategory.module.scss";
+import productIcon from "./Icons/Icons.png";

```

Comment:
there is only one icon. why do we name it "icons"? 
please be more specific, as UX guys for a better name, if needed ;)
and also prefer using svg over png.